docs/content/preview/api/ysql/exprs/func_yb_is_local_table.md

```diff
@@ -0,0 +1,224 @@
+---
+title: yb_is_local_table() function [YSQL]
+headerTitle: yb_is_local_table(oid)
+linkTitle: yb_is_local_table()
+description: Returns whether the given 'oid' is a table replicated only in the local region.
+menu:
+  preview:
+    identifier: api-ysql-exprs-yb_is_local_table
+    parent: api-ysql-exprs
+type: docs
+---
+
+## Synopsis
+
+`yb_is_local_table` is a function used to find whether a table is pinned to the local region. It takes the OID (Object Identifier) of the table as an argument and looks for a tablespace associated with the table. It checks whether the tablespace is confined to the same region as that of the PostgreSQL backend processing the query. If so, it returns true, false otherwise.
```

Comment:
Is this actually the _YugabyteDB backend_ here?

```suggestion
`yb_is_local_table` indicates whether a table is pinned to the local region. The function takes a table's OID (object identifier), and looks for a tablespace associated with that table. If the tablespace is confined to the same region as that of the PostgreSQL backend processing the query, the function returns `true`. Otherwise, the function returns `false`.
```